demo/CMakeLists.txt

```diff
@@ -22,6 +22,9 @@
   SOFTWARE.
 ]]
 
+set(CMAKE_AUTOMOC true)
```

Comment:
Better change true to ON

Reply:
I noticed that it's actually not needed at all, since it's in the top level CMakeLists. Will remove this line